.github/workflows/build.yml

```diff
@@ -98,25 +99,75 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 16.x
-      
+
       - name: Build bicep-types dependency
         run: |
           npm ci
           npm run build
         working-directory: bicep-types/src/bicep-types
-      
+
       - name: Install
         run: npm ci
         working-directory: ./src/autorest.bicep
-      
+
       - name: Lint
         run: npm run lint
         working-directory: ./src/autorest.bicep
 
       - name: Build
         run: npm run build
         working-directory: ./src/autorest.bicep
+
+      - name: Test
+        run: npm test
+        working-directory: ./src/autorest.bicep
+
+  test-github-scripts:
```

Comment:
Would it be better to move this job to a separate workflow and only trigger it upon script changes?

Reply:
This is just a test, why not run it every time though?